process/block/metablock.go

```diff
@@ -202,6 +203,18 @@ func (mp *metaProcessor) ProcessBlock(
 		return err
 	}
 
+	validatorStatsRH, err := mp.validatorStatisticsProcessor.UpdatePeerState(header)
+	if err != nil {
+		log.Error("cannot update peer state")
+		return err
+	}
+
+	log.Error(fmt.Sprintf("%s %s", core.ToB64(header.ValidatorStatsRootHash), core.ToB64(validatorStatsRH)))
```

Comment:
remove the error log? it is not really an error

Reply:
Yes, removed, it was a forgotten debug print